### ISSUE TITLE
Mac: difficult to close picture after opening it

### ISSUE THREAD

**Issue by skyfaller** [ShihabMe/Mast2#5]
When I click on a picture in my timeline, it opens an attractive-looking view of the image that covers the whole app. Unfortunately, there is no "x" to close the image, and hitting the Escape key does nothing. It took me a while to discover the one way to close the image without quitting the app: click on the description of the image at the bottom of the screen. Even this is not ideal, because it opens a "detail" column, replacing my profile column, which would be fine if there were a way to close the image without doing that, but it shouldn't be the only option.

TL;DR: Please provide more (obvious) ways to leave the image view.
